Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1001,6 +1001,8 @@ namespace Multiplayer
 
     void MultiplayerSystemComponent::InitializeMultiplayer(MultiplayerAgentType multiplayerType)
     {
+        m_lastReplicatedHostFrameId = HostFrameId{0};
```

Comment:
Please ignore this chance, I am on a development branch behind by a few commits due to a known issue in dev. This change is already in dev.